smithy-aws-protocol-tests/model/awsJson1_1/null.smithy

```diff
@@ -33,7 +33,10 @@ use smithy.test#httpResponseTests
                 "string": null
             }""",
         bodyMediaType: "application/json",
-        headers: {"Content-Type": "application/x-amz-json-1.1"},
+        headers: {
+            "Content-Type": "application/x-amz-json-1.1",
+            "X-Amz-Target": "JsonProtocol.NullOperation",
```

Comment:
There's a test above this that, while `client` specific, should have the header.

Reply:
Thank you. Updated. However, I don't have any test at hand to verify their behaviour.